addons/website/controllers/main.py

```diff
@@ -333,6 +333,14 @@ def get_dynamic_snippet_templates(self, filter_name=False):
         templates = request.env['ir.ui.view'].sudo().search_read(domain, ['key', 'name'])
         return templates
 
+    @http.route('/website/get_current_currency', type='json', auth="public", website=True)
+    def get_current_currency(self, **kwargs):
+        return {
+            'id': request.website.company_id.currency_id.id,
+            'symbol': request.website.company_id.currency_id.symbol,
+            'position': request.website.company_id.currency_id.position,
+        }
+
```

Comment:
It feels like this belongs to the session_info dict, and that would also avoid extra useless requests, no?

Reply:
not :/

`odoo.__session_info__ = {"is_admin": false, "is_system": false, "is_website_user": true, "user_id": false, "is_frontend": true, "profile_session": null, "profile_collectors": null, "profile_params": null, "translationURL": "/website/translations", "cache_hashes": {"translations": "63c39b9719623b23090242bff39258f3aa29fe2b"}};`

maybe we could move it into session ? but on all pages in this case and not only when you have the snippet.
 